Source/WebCore/loader/ContentFilter.cpp

```diff
@@ -123,6 +123,73 @@ bool ContentFilter::continueAfterWillSendRequest(ResourceRequest& request, const
     return !request.isNull();
 }
 
+ContentFilter::ContentFilterCallbackAggregator::~ContentFilterCallbackAggregator()
+{
+    ASSERT(m_callback);
+    CheckedPtr contentFilter = m_contentFilter;
```

Comment:
We should reduce the scope of this variable to not cover the `m_callback` calls. It is too risky otherwise given that this is a CheckedPtr.